pl_bolts/models/autoencoders/basic_vae/basic_vae_module.py

```diff
@@ -25,7 +25,7 @@ def __init__(
             batch_size: int = 32,
             learning_rate: float = 0.001,
             data_dir: str = '.',
-            datamodule: pl_bolts.datamodules.LightningDataModule = None,
+            datamodule: LightningDataModule = None,
```

Comment:
@nateraw does trainer.fit(ae) automatically accept self.datamodule in this case? I can't see a call to trainer.fit(ae, datamodule)